plugins/addAttributesToSVGElement.js

```diff
@@ -1,6 +1,29 @@
+/**
+ * @typedef {import('json-schema-typed').JSONSchema} JSONSchema
+ */
+
 export const name = 'addAttributesToSVGElement';
 export const description = 'adds attributes to an outer <svg> element';
 
+/** @type {JSONSchema} */
+export const schema = {
+  type: 'object',
+  required: ['attributes'],
+  properties: {
+    attributes: {
+      title: 'Attributes',
+      description:
+        'Attributes to add to the `<svg>` element. If key/value pairs are passed, the attributes are added with the paired value. If an array is passed, attributes are added with no key associated with them.',
```

Comment:
A lot of this description only applies to the JS API, and wouldn't apply to the UI. I'm not sure what to do about that, because the docs need the JS details.